src/wdio/services/TerraCommands/hide-input-caret.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Hides the blinking input caret that appears within editable text areas to prevent inconsistent test failures.
+ *
+ * @param {string} selector The selector for the element to hide the caret of
+ */
+const hideInputCaret = (selector) => {
```

Comment:
What if we set this on `#root ` after any url or refresh command?
We'd have to make sure it works, but do we ever want the input caret to be visible for wdio tests?

Reply:
It looks like this does get inherited, so this is a possibility. I can't think of any reason someone would want the caret to be visible.

Reply:
Talked in scrum, seems we're good to go for this

Reply:
Does this need updated for a fallback then?
```
const hideInputCaret = (selector = 'body') => {

or 

const hideInputCaret = (selector = '#root') => {
```
to allow for
```
Terra. hideInputCaret()
```

or is the afterCommand call good enough?

Reply:
The `afterCommand` call should be good enough. If someone has code to change the `caret-color` on the body element when their component loads then they can call `Terra.hideInputCaret('body');`.